x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/entity_filter/entity_filter.tsx

```diff
@@ -22,64 +22,70 @@ interface EntityFilterProps {
   }) => void;
   influencerFieldName: string;
   influencerFieldValue: string;
+  mode?: string;
 }
 export const EntityFilter: FC<EntityFilterProps> = ({
   onFilter,
   influencerFieldName,
   influencerFieldValue,
+  mode,
 }) => {
   return (
-    <React.Fragment>
-      <EuiToolTip
-        content={
-          <FormattedMessage
-            id="xpack.ml.entityFilter.addFilterTooltip"
-            defaultMessage="Add filter"
+    <EuiFlexGroup gutterSize="none" alignItems="center">
+      <EuiFlexItem grow={false}>
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="xpack.ml.entityFilter.addFilterTooltip"
+              defaultMessage="Filter for"
+            />
+          }
+        >
+          <EuiButtonIcon
+            size="s"
+            className="filter-button"
+            onClick={blurButtonOnClick(() => {
+              onFilter({
+                influencerFieldName,
+                influencerFieldValue,
+                action: ML_ENTITY_FIELD_OPERATIONS.ADD,
+              });
+            })}
+            iconType="plusInCircle"
+            aria-label={i18n.translate('xpack.ml.entityFilter.addFilterAriaLabel', {
+              defaultMessage: 'Add filter for {influencerFieldName} {influencerFieldValue}',
+              values: { influencerFieldName, influencerFieldValue },
+            })}
           />
-        }
-      >
-        <EuiButtonIcon
-          size="s"
-          className="filter-button"
-          onClick={blurButtonOnClick(() => {
-            onFilter({
-              influencerFieldName,
-              influencerFieldValue,
-              action: ML_ENTITY_FIELD_OPERATIONS.ADD,
-            });
-          })}
-          iconType="plusInCircle"
-          aria-label={i18n.translate('xpack.ml.entityFilter.addFilterAriaLabel', {
-            defaultMessage: 'Add filter for {influencerFieldName} {influencerFieldValue}',
-            values: { influencerFieldName, influencerFieldValue },
-          })}
-        />
-      </EuiToolTip>
-      <EuiToolTip
-        content={
-          <FormattedMessage
-            id="xpack.ml.entityFilter.removeFilterTooltip"
-            defaultMessage="Remove filter"
+        </EuiToolTip>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiToolTip
+          content={
+            <FormattedMessage
+              id="xpack.ml.entityFilter.removeFilterTooltip"
+              defaultMessage={mode ? 'Filter out' : 'Remove filter'}
```

Comment:
This will need two different translations. There can't be two possible values for one translation ID

Reply:
Good catch - updated in 41c37ae127f9a44b4f83250e2b3fe1def6953a11